node-tests/blueprints/eslint-config-simplabs-test.js

```diff
@@ -47,7 +52,8 @@ describe('eslint-config-simplabs blueprint', function() {
       .then(() => emberGenerate(['eslint-config-simplabs']))
       .then(() => {
         expect(file('.eslintrc.js')).to.contain('extends: \'simplabs\'');
-        expect(file('tests/.eslintrc.js')).to.equal(`module.exports = {\n  extends: 'simplabs/configs/ember-mocha',\n};\n`);
+        expect(file('tests/.eslintrc.js'))
+          .to.equal(`module.exports = {\n  extends: [\n    'simplabs/configs/ember-mocha',\n    'simplabs/rules/mocha',\n  ],\n};\n`);
```

Comment:
P.S. template string syntax lets you use multi-line strings. Unfortunately it doesn't play well with indenting, but you could do something like:
```js
const assertedValue = `{
  extends: [ ... ]
}`
```